### PR TITLE
Document `Gil_syntax`

### PR DESCRIPTION
**Pull request by NatKarmios**

### CODE REVIEW

File: GillianCore/GIL_Syntax/Constant.ml [hunk 1]
```diff
@@ -5,10 +5,10 @@
 type t = TypeDef__.constant =
   | Min_float  (** The smallest float *)
   | Max_float  (** The largest float *)
-  | MaxSafeInteger  (** {% 2^53 - 1 %} **)
+  | MaxSafeInteger  (** [2^53 - 1] *)
   | Epsilon  (** Smallest positive number *)
   | Random  (** A random number between 0 and 1 *)
-  | Pi  (** The number {% \pi %} *)
+  | Pi  (** The number [pi] *)
```
Comment (giltho) [inline GillianCore/GIL_Syntax/Constant.ml:11]: Wouldn't we rather have `{m \pi}` ?

Comment (giltho) [reply GillianCore/GIL_Syntax/Constant.ml:11]: Applies to `{m 2^53 - 1}`

**Review by giltho — APPROVED** [review]

File: GillianCore/GIL_Syntax/Constant.ml (hunk 1 above)
Comment (NatKarmios) [reply GillianCore/GIL_Syntax/Constant.ml:11]: I noticed the `{m }` syntax in the odoc docs about an hour after pushing this :P

Comment (giltho) [reply GillianCore/GIL_Syntax/Constant.ml:11]: Wanna have fun? Look at who wrote the PR which introduces it :p

**Review by giltho — DISMISSED** [review]

**Review by giltho — APPROVED** [review]

**Pull request merged**